src/module.rs

```diff
@@ -120,6 +121,14 @@ impl<'a> Module<'a> {
             .all(|segment| segment.value.is_empty())
     }
 
+    /// Get values of the module's segments
+    pub fn get_segments_width(&self) -> usize {
+        self.segments
+            .iter()
+            .map(|segment| segment.value.width())
```

Comment:
Should this use `print::better_width()`?

Reply:
`print::better_width()` doesn't seem to exist anymore, so I'm assuming that `UnicodeWidthGraphemes::width_graphemes()` is the preferred approach, and will update to use that.